README.md

```diff
@@ -1 +1,170 @@
-# hello-react-front-end
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+<h1><b>hello-react-front-end</b></h1>
+ 
+  <img src="./page1.png" alt="logo" width="200"  height="auto">
+  <img src="./page2.png" alt="logo" width="200"  height="auto">
+  <img src="./erd_diagram.png" alt="logo" width="200"  height="auto">
+  <br/>
```

Comment:
- [ ] It would be nice if the logo in your readme is updated, currently this shows as a broken image, please update it  👍🏿